.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules
+coverage
+.DS_Store
```

Comment:
`.DS_Store` should not be in here.

If you want to ignore those system files, [do it globally](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).